docker/package/tezos_voting_wizard.py

```diff
@@ -125,6 +119,44 @@ def get_proposal_period_hash(hashes):
 )
 
 
+def get_node_rpc_addr_query(default=None):
```

Comment:
There doesn't seem to be a reliable source of public nodes, but perhaps we can still give multiple choice (per network) that we know are working now and one to input it manually (like now).

This would give a default option that's good enough in most cases. What do you think? 

P.s. I'm ok with opening a separate issue for this.